test/prism_regression/assign_to_constant.rb

```diff
@@ -73,9 +78,13 @@ def method1
 end
 
 def method2
-  ConstantPath::DynamicConstant2 = "This should raise a SyntaxError at runtime"
+  # * `ruby -c --parser=parse.y` reports dynamic constant assignments for all of them.
+  # * `ruby -c --parser=prism`   only reports the first one.
+  # * At runtime, Ruby will issue warnings for all of these.
+
+  ConstantPath::DynamicConstant2 = 1
 
-  # These should *NOT* raise a SyntaxError at runtime
+  # Sorbet's previous parser would treat all of these as dynamic constant assignments.
```

Comment:
Does `all` here means all of the belows? If that's the case, can we phrase it that way? Just to make it clearer whether the 1st case is included or now.